tools/SourceKit/lib/SwiftLang/SwiftEditor.cpp

```diff
@@ -151,6 +151,8 @@ void EditorDiagConsumer::handleDiagnostic(
       break;
     case DiagnosticKind::Note:
       llvm_unreachable("already covered");
+    case DiagnosticKind::Remark:
+      llvm_unreachable("remarks unsupported");
```

Comment:
Trapping here isn't acceptable. @akyrtzi, @nkcsgexi?

Reply:
Yes, we should not cause a crash, I recommend checking for them at the beginning and:
- Ignore them and log that there was occurrence of remark that we ignored, see LOG_WARN_FUNC that is used elsewhere in the file.
- Add a FIXME to consider if there is a need to support them from sourcekitd